modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -109,7 +110,6 @@ public Swift5ClientCodegen() {
                         "Bool",
                         "Void",
                         "String",
-                        "URL",
```

Comment:
Why did you dele URL here?

Reply:
duped in line [#117](https://github.com/OpenAPITools/openapi-generator/pull/8515/files#diff-4a35291cd15aa856298c1f88a25af4ed1043f7776bff67e3803f20008a7763a6R117)

Reply:
I added a comment on line 217